.gitignore

```diff
@@ -0,0 +1 @@
+clipse
```

Comment:
It's strange to see this committed, ay least with changes that are unrelated

Reply:
Oh yes, sorry I should have mentioned that... I thought it would be useful as anyone contributing would be building clipse regularly and it would be a good convenience to have